src/main/java/org/dataloader/registries/ScheduledDataLoaderRegistry.java

```diff
@@ -57,6 +61,86 @@ public Duration getScheduleDuration() {
         return schedule;
     }
 
+    /**
+     * This will combine all the current data loaders in this registry and all the data loaders from the specified registry
+     * and return a new combined registry
+     *
+     * @param registry the registry to combine into this registry
+     *
+     * @return a new combined registry
+     */
+    public ScheduledDataLoaderRegistry combine(DataLoaderRegistry registry) {
+        Builder combinedBuilder = ScheduledDataLoaderRegistry.newScheduledRegistry()
+                .dispatchPredicate(this.dispatchPredicate);
+        combinedBuilder.registerAll(this);
+        combinedBuilder.registerAll(registry);
+        return combinedBuilder.build();
+    }
+
+
+    /**
+     * This will unregister a new dataloader
+     *
+     * @param key the key of the data loader to unregister
+     *
+     * @return this registry
+     */
+    public ScheduledDataLoaderRegistry unregister(String key) {
+        DataLoader<?, ?> dataLoader = dataLoaders.remove(key);
+        if (dataLoader != null) {
+            dataLoaderPredicates.remove(dataLoader);
+        }
+        return this;
+    }
+
+    /**
+     * @return the current dispatch predicate
+     */
+    public DispatchPredicate getDispatchPredicate() {
+        return dispatchPredicate;
+    }
```

Comment:
Now that a `DispatchPredicate` can be registered per `DataLoader`, this should ideally be called `defaultDispatchPredicate`. However, I understand that would involve renaming, and/or a deprecation and a property with the new name. In the very least, the Javadoc should be updated to indicate that it applies by default, if there isn't already a registration for that `DataLoader`. 

Also, really minor but field and getter could be ordered after the map of predicates like they are in the builder since this is now a much less central property and rather optional.

Reply:
done.  I didnt deprecate but I did redocument